public/index.html

```diff
@@ -7,7 +7,7 @@
       window.ENV = {
         REACT_APP_COUNTRY_RISK_API: "http://localhost:8080",
         REACT_APP_AUTH_URL: "http://localhost:8080/auth",
-        REACT_APP_PORTAL_FRONTEND: "http://localhost:8080",
+        REACT_APP_PORTAL_FRONTEND: "http://localhost:3000",
```

Comment:
not sure what this change has to do with the about page, but if you want to do it, I suggest to replace the anchor [here](https://github.com/eclipse-tractusx/vas-country-risk/blob/f90173fbb9b611b632342e531d06114709e2bc1c/scripts/inject-dynamic-env.sh#L29) as well, other wise the dynamic env injection won't work for this value.